google/cloud/pubsub_v1/subscriber/message.py

```diff
@@ -322,6 +332,12 @@ def ack_with_response(self) -> "futures.Future":
             pubsub_v1.subscriber.exceptions.AcknowledgeError exception
             will be thrown.
         """
+        _ACK_NACK_LOGGER.debug(
+            "Called ack for message (id=%s, ack_id=%s, ordering_key=%s, exactly_once=True)",
```

Comment:
Could users still use the ack_with_response interface when EOD is disabled? If so, it would be worth setting `exactly_once=self.exactly_once_enabled`